thrift/contentatom.thrift

```diff
@@ -0,0 +1,5 @@
+include "atoms/tenfour.thrift"
+
+union ContentAtom {
+  1: tenfour.Content tenfour
```

Comment:
Let me just state my complete ignorance by asking shamelessly: what does ten four stand for or refer to in the first place? :pensive: 

Possibly the fact that current quizzes are typically 10 questions with 4 answers...?